utils/action.js

```diff
@@ -15,5 +15,7 @@ module.exports.logAction = async function(user, key, details) {
                 ? JSON.stringify(details)
                 : details
     });
-    return action.setUser(user);
+    action.setUser(user);
```

Comment:
Is this returning a promise too and needs to be `await`ed?

Reply:
nope. the `setFoo()` methods in a model are sync because they aren't modifying the db until the `model.save()` call

Reply:
Hmm, here they do `project.setTasks` and call `then` on it. 🤷🏻‍♂️
https://sequelize.org/master/manual/associations.html#associating-objects

Reply:
I think that's because it's not a field but a relation

Reply:
Is it actually documented somewhere in the sequelize docs? I always have trouble finding infos about these magic setters and getters.

Reply:
I could have also used 
```js
action.user = user;
```
but I was unsure if that would work with both `userId` and `user` model.
see https://sequelize.org/master/manual/instances.html#updating---saving---persisting-an-instance

Reply:
I see. If it works sure. I think I would have associated the user directly in `Action.create`. Something like this:
```js
await Action.create({ user_id: userId, ... })
```